cypress/e2e/features/Ui/size.feature

```diff
@@ -9,16 +14,34 @@ Feature: Size
         When User clicks to size tab
         And User chooses "<size>" from list
         Then User verifies the result page cantains only items with "<size>"
+    Examples:
+    | size| 
+    | 18  | 
+    | 24  | 
+    | 30  |
+    | 32  | 
+    | 36  | 
+    | 40  |
+    | 48  | 
+    | 60  | 
+    | 67  | 
+    | 72  |
+
 
-        Examples:
-            | size |
-            | 18   |
-            | 24   |
-            | 30   |
-            | 32   |
-            | 36   |
-            | 40   |
-            | 48   |
-            | 60   |
-            | 67   |
-            | 72   |
+ 
+    Scenario Outline: Size Navigation Check
+    When User clicks to size tab 
+    And User chooses "<size>" from list
+    Then User verifies the result page url contains "<size>"
```

Comment:
What about removing this scenario and adding line 35 to line 17? No need to increase execution time if it is possible to verify with one scenario.